### PR TITLE
f-feature/gh-315-search-users-from-header

### PR DESCRIPTION
**Pull request by aberonja**
Added user search.

GH issue: #315

### CODE REVIEW

File: src/main/frontend/src/components/search-dropdown/search-results/EmployeeSearchResults.js [hunk 1]
```diff
@@ -1,8 +1,30 @@
...
 import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { NavLink } from 'react-router-dom';
+import { bindActionCreators } from 'redux';
+import { addSearchedEmployee } from '../../../store/actions/employeesSearchActions';
 import Loading from '../../loading/Loading';
 
-export default class EmployeeSearchResults extends Component {
+class EmployeeTeamSearchResults extends Component {
+  // TODO: Refactor this!!!
```
Comment (iscekic) [inline src/main/frontend/src/components/search-dropdown/search-results/EmployeeSearchResults.js]: !!!

Comment (iscekic) [reply src/main/frontend/src/components/search-dropdown/search-results/EmployeeSearchResults.js]: Just use relative routes. 😉

File: src/main/frontend/src/store/reducers/leaveRequestsReducer.js [hunk 2]
```diff
@@ -1,25 +1,27 @@
...
 import { navigateOut } from '../actions/navigateActions';
-import { receiveLeaveRequestsForYear } from '../actions/requestsActions';
+import {
+  receiveLeaveRequestsForYear,
+  receiveUserLeaveRequestsForYear
+} from '../actions/requestsActions';
 
 export const initialState = null;
 
+const makeFilterUpdaterReducer = () => (state, action) => ({
```
Comment (iscekic) [inline src/main/frontend/src/store/reducers/leaveRequestsReducer.js]: No need to wrap this in a function.

File: src/main/frontend/src/store/reducers/reducerHelpers.js [hunk 3]
```diff
@@ -0,0 +1,4 @@
+export const makeUpdaterReducer = propName => (state, action) => ({
```
Comment (iscekic) [inline src/main/frontend/src/store/reducers/reducerHelpers.js]: 💃

Comment (aberonja) [reply src/main/frontend/src/store/reducers/reducerHelpers.js]: 🐑 👅

**Pull request merged**